webview/src/experiments/util/buildDynamicColumns.tsx

```diff
@@ -38,17 +42,21 @@ const CellTooltip: React.FC<{
   )
 }
 
-const Cell: React.FC<Cell<Experiment, string | number>> = cell => {
+const Cell: React.FC<
+  Cell<Experiment, string | number | ValueWithChanges>
+> = cell => {
   const { value } = cell
   if (value === undefined) {
     return UndefinedCell
   }
 
-  const stringValue = String(value)
+  const rawValue = (value as ValueWithChanges).value ?? value
```

Comment:
[I] Would be good to have a couple of helper functions for 

```
const rawValue = (value as ValueWithChanges).value ?? value
const cellhasChanges = (cell.value as ValueWithChanges)?.changes
```

so that we can keep them together and have the concept all in one place (easier to extend/reason about later)